rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -1078,6 +1078,9 @@ enum Method
 
 	RSX_METHOD_NOP_CMD = 0x00000000,
 	RSX_METHOD_NOP_MASK = 0xbfff0003,
+
+	// Stack is empty (invalid value)
+	RSX_CALL_STACK_EMPTY = 3,
```

Comment:
Cosmetic, but just add the padding 0s for consistency with the previous enumerants.